spec/druid_spec.rb

```diff
@@ -1,15 +1,12 @@
 describe DruidTools::Druid do
-  before (:all) do
-    @fixture_dir = File.expand_path("../fixtures", __FILE__)
-    FileUtils.rm_rf(File.join(@fixture_dir, 'cd'))
-    @druid_1 = 'druid:cd456ef7890'
-    @tree_1 = File.join(@fixture_dir, 'cd/456/ef/7890/cd456ef7890')
-    @druid_2 = 'druid:cd456gh1234'
-    @tree_2 = File.join(@fixture_dir, 'cd/456/gh/1234/cd456gh1234')
-  end
+  let(:fixture_dir) { File.expand_path("../fixtures", __FILE__) }
+  let(:valid_druid) { 'druid:cd456ef7890' }
```

Comment:
`loosely_valid_druid`?  `lax_valid_druid`?  `almost_valid_druid`?  😛 

(fine with it staying as-is, but it might be slightly confusing/misleading at first glance if the name is left unqualified)